### PR TITLE
Fix #127

### LINKED ISSUE #127: Does not fully close when using gesture to close menu.
When using a pan gesture to close the menu, it gets stuck before fully closing. See the gif below:
side-menu-issue

Messing around with code, I'm able to fix this by removing the if statement in `handlePanResponderMove`: 
```
if (this.menuPositionMultiplier() *
      (this.state.left.__getValue() + gestureState.dx ) >= 0 ) {
      this.state.left.setValue(this.prevLeft + gestureState.dx);
    }
```
And instead just call:
`this.state.left.setValue(this.prevLeft + gestureState.dx);`

It seems to work, and `handleMoveShouldSetPanResponder` seems to weed out any unwanted pans. But I bet it breaks something else. 

Can someone familiar with this project explain what that `if` statement is for?

### PR DESCRIPTION
**Pull request by Kureev**
Simplified `if` condition on pan responder move.

Can you review, @grabbou?

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -137,9 +137,7 @@ class SideMenu extends Component {
    * @return {Void}
    */
   handlePanResponderMove(e: Object, gestureState: Object) {
-    if (this.menuPositionMultiplier() *
```
Comment (grabbou) [inline index.js]: That's pretty hard in reasoning statement to be fair :dancer:

Comment (Kureev) [inline index.js]: m?

Comment (grabbou) [inline index.js]: Nothing, just had the `AHA` moment when going through it.

Comment (Kureev) [inline index.js]: :laughing:

**Comment by grabbou** [comment]
As mentioned on Discord, that will make the right-sided menu not working, because in that case, `left` will become negative straight away.

File: index.js (hunk 1 above)
Comment (Kureev) [inline index.js]: Corrected

**Comment by grabbou** [comment]
@Kureev now it should be good.

**Pull request merged**